libraries/clickhouse-helpers/clickhouse_helpers/execute_sql/execute_sql.py

```diff
@@ -2,7 +2,6 @@
 
 import pandas as pd
 import ramda as R
-from pandahouse import to_clickhouse
```

Comment:
Turns out, pandahouse is unable to handle None values in integer columns.

Reply:
why are there None values anyways? i noticed that in the daily vital data that there were some for days that technically we shouldnt have data for (i.e. before the study officially started on 4/12/2020). is it similar?